src/ryr-preview.html

```diff
@@ -5,6 +5,7 @@
     <style>
       :host {
         display: block;
+        color: black;
```

Comment:
Thinking of it a bit more, this should be a custom CSS property (possibly defaulting to `black`): `color: var(--preview-text-color, black);`

See here for more details: https://www.polymer-project.org/2.0/docs/devguide/custom-css-properties#create-custom-properties

Reply:
@rgreinho - Got it. Working on it now. Will fix it. Thanks!